helm/values-prod.yaml

```diff
@@ -0,0 +1,164 @@
+    initialDelaySeconds: 40
+    failureThreshold: 6
+    periodSeconds: 10
+  readinessProbe:
+    httpGet:
+      path: /actuator/health/readiness
+      port: 8080
+    initialDelaySeconds: 40
+    failureThreshold: 6
+    periodSeconds: 10
+  deployment:
+    create: true
+    replicas: 3
+  service:
+    create: true
+    type: ClusterIP
+    ports:
+      - 8080
+  ingress:
+    create: true
+    host: "itnprod.pay-wallet.internal.platform.pagopa.it"
+    path: /pagopa-payment-wallet-scheduler-service/(.*)
+    servicePort: 8080
+  serviceAccount:
+    create: false
+    annotations: { }
+    name: ""
+  podAnnotations: { }
+  podSecurityContext:
+    seccompProfile:
+      type: RuntimeDefault
+  securityContext:
+    allowPrivilegeEscalation: false
+  resources:
+    requests:
+      memory: "512Mi"
+      cpu: "300m"
+    limits:
+      memory: "512Mi"
+      cpu: "600m"
+  autoscaling:
+    enable: false
+    minReplica: 3
+    maxReplica: 3
+    pollingInterval: 10 # seconds
+    cooldownPeriod: 50 # seconds
+    triggers:
+      - type: cpu
+        metadata:
+          # Required
+          type: Utilization # Allowed types are 'Utilization' or 'AverageValue'
+          value: "75"
+  terminationGracePeriodSeconds: 30
+  strategy:
+    type: "RollingUpdate"
+    rollingUpdate:
+      maxSurge: "100%"
+      maxUnavailable: "100%"
+  envConfig:
+    OTEL_SERVICE_NAME: "pagopa-payment-wallet-scheduler-service"
+    OTEL_SERVICE_ENVIRONMENT: "prod"
+    OTEL_RESOURCE_ATTRIBUTES: "deployment.environment=prod"
+    OTEL_EXPORTER_OTLP_ENDPOINT: "https://weuprod.kibana.internal.platform.pagopa.it/apm"
+    OTEL_EXPORTER_OTLP_PROTOCOL: http/protobuf
+    OTEL_TRACES_EXPORTER: otlp
+    OTEL_METRICS_EXPORTER: otlp
+    OTEL_LOGS_EXPORTER: otlp
+    OTEL_TRACES_SAMPLER: "always_on"
+    MONGO_HOST: pagopa-p-itn-pay-wallet-cosmos-account.mongo.cosmos.azure.com
+    MONGO_USERNAME: pagopa-p-itn-pay-wallet-cosmos-account
+    MONGO_PORT: "10255"
+    MONGO_SSL_ENABLED: "true"
+    MONGO_DB_NAME: "payment-wallet"
+    MONGO_MIN_POOL_SIZE: "0"
+    MONGO_MAX_POOL_SIZE: "50"
+    MONGO_MAX_IDLE_TIMEOUT_MS: "600000"
+    MONGO_CONNECTION_TIMEOUT_MS: "2000"
+    MONGO_SOCKET_TIMEOUT_MS: "10000"
+    MONGO_SERVER_SELECTION_TIMEOUT_MS: "2000"
+    MONGO_WAITING_QUEUE_MS: "2000"
+    MONGO_HEARTBEAT_FREQUENCY_MS: "5000"
+    MONGO_REPLICA_SET_OPTION: "&replicaSet=globaldb"
+    SCHEDULER_CDC_QUEUE_NAME: pagopa-p-itn-pay-wallet-cdc-queue
+    SCHEDULER_CDC_QUEUE_TTL_SECONDS: "604800" #7 days
+    SCHEDULER_CDC_QUEUE_VISIBILITY_TIMEOUT_SECONDS: "0"
+    SCHEDULER_CDC_SEND_RETRY_MAX_ATTEMPTS: "1"
+    SCHEDULER_CDC_SEND_RETRY_INTERVAL_IN_MS: "1000"
+    WALLET_SEARCH_STATUS: "VALIDATED"
+    WALLET_SEARCH_LIMIT: "20"
+    SCHEDULER_REDIS_RESUME_KEYSPACE: "pay-wallet-scheduler"
+    SCHEDULER_REDIS_RESUME_TTL_IN_MIN: "2880" #2 days
+    SCHEDULER_REDIS_JOB_LOCK_KEYSPACE: "pay-wallet-scheduler-locks"
+    SCHEDULER_REDIS_JOB_LOCK_TTL_MS: "20000"
```

Comment:
I would set the lock TTL to 30 sec to avoid overlapping during the first iterations due to warmup.